install_debian_autopsy_4.19.2.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+sudo apt install wget
```

Comment:
All the calls, except where resources are fetched, are run under `root`. Sudo has a timeout loop, for safety reasons, and therefore this script can be hard to automate as is, since timeout may reach and password is asked again. I would suggest checking if the user id is 0, otherwise halt it. And the user should call the script with `sudo` itself.

Reply:
That will download the files under root, so they will be owned by root, which can be an issue if they are not to be deleted. I guess not too big of an issue.